### PR TITLE
allow running debugtopus environment-aware

### PR DESCRIPTION
**Pull request by Germandrummer92**
adds new OPTIONAL parameter that will fetch the code for the specified environment

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,17 +28,17 @@ You should just add the url of the app you would like to run the test against:
 
 Running one test case:
 ```shell
-npx @octomind/debugtopus --id <test-case-id> --token <token> --url <local-url> --testTargetid <testTargetid>
+npx @octomind/debugtopus --id <testCaseId> --token <token> --url <local-url> --testTargetId <testTargetId> --environmentId <environmentId>
```
Comment (kosta) [inline README.md:31]: What about `local-url`?

File: README.md [hunk 2]
```diff
@@ -27,17 +28,17 @@ You should just add the url of the app you would like to run the test against:
...
 Running all test cases in a test target:
 ```shell
-npx @octomind/debugtopus --token <token> --url <local-url> --testTargetid <testTargetid>
+npx @octomind/debugtopus --token <token> --url <local-url> --testTargetId <testTargetId> --environmentId <environmentId>
 ```
 
 You can also use a specific version:
 ```shell
- npx --package @octomind/debugtopus@<some-version> debugtopus --id <test-case-id> --token <token> --url <local-url> --testTargetid <testTargetid>
+ npx --package @octomind/debugtopus@<some-version> debugtopus --id <test-case-id> --token <token> --url <local-url> --testTargetid <testTargetid> --environmentId <environmentId>
```
Comment (kosta) [inline README.md:41]: Did you give up on converting kebap-case to camelCase half-way through the readme?

**Review by kosta — APPROVED** [review]
Not asked but I approve

File: README.md (hunk 1 above)
Comment (Germandrummer92) [reply README.md:31]: oopsie :)

File: README.md (hunk 2 above)
Comment (Germandrummer92) [reply README.md]: fixed!

**Pull request merged**